framework/caching-cmm.md

```diff
@@ -149,3 +162,102 @@ the caching CMM MUST NOT store the decryption materials in the underlying CMC.
 ## Security Considerations
 
 [TODO: What security properties can the caching CMM guarantee?]
```

Comment:
This might be outside of the scope of this PR, but we *really* need to fix this. We shouldn't have a Spec 1.0 where we have TODO's in the spec.